src/qml/EmbeddedFeatureForm.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+
+import QtQuick.Layouts 1.0
+import "."
+import Theme 1.0
+
+import org.qfield 1.0
+import org.qgis 1.0
+
+Popup {
+    id: formPopup
+
+    //the add entry stuff
+    property var attributeFormModel: AttributeFormModel {
```

Comment:
Is this property

- expected to be set from outside (in this case, do we need the default value in here)
- or only for reading (in this case should the property be readonly)
- or not for outside usage (in this case should it be in an embedded QtObject{})

Reply:
expected to be set from the outside, but we do not need to set properties of the AttributeFormModel but of it's FeatureModel. That's why the default value.

Reply:
Then do the model internally and add an alias (proxy) property https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html

Reply:
I don't know how to work with an alias here. 

When I do `property alias featureModel:form.model.featureModel` and connect it from outside, and set e.g. the `embeddedForm.featureModel.currentLayer` it says that it cannot set a property of null. Of course, FeatureModel is not created.

So I see the possibility to make an alias for every FeatureModel-Property (currentLayer, feature etc.) or to create the FeatureModel from outside and set it via `alias`. I don't like these solutions.

But maybe I don't see the obvious.

Reply:
I did it now with aliases of the featureModel properties.